src/js/views/patients/worklist/worklist_views.js

```diff
@@ -30,7 +32,7 @@ const EmptyView = View.extend({
   `,
 });
```

Comment:
we'll need to update the `EmptyView` too to display the proper text (No Actions vs No Flows)

Reply:
Ah thank you. That was on my radar then slipped my mind